pkg/lang/golang/imports.go

```diff
@@ -47,6 +47,16 @@ func GetImportsInFile(f *core.SourceFile) []Import {
 	return imports
 }
 
+func GetNamedImportInFile(f *core.SourceFile, namedImport string) Import {
+	imports := GetImportsInFile(f)
+	for _, i := range imports {
```

Comment:
minor-ish: I pretty much always assume `i` is an index in a `for`, so this threw me off at first. Maybe `imp`?

Reply:
yeah because i have changes in other prs i will go ahead and update in a future pr. Same with the above comments